getting_started.ipynb

```diff
@@ -0,0 +1,1962 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 15,
+   "id": "04a11bb8-e4d9-4687-ab89-210aa1b7b51b",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from langchain_core.prompts import PromptTemplate  \n",
+    "\n",
+    "prompt_template = PromptTemplate.from_template(\n",
+    "    \"What vege crops can I grow in {month} in {city}, New Zealand?\"\n",
+    ")\n",
+    "\n",
+    "prompt_value = prompt_template.format(month = \"December\", city = \"Rotorua\")\n",
+    "\n",
+    "# print(prompt_template) # <- uncomment to see\n",
+    "# print(prompt_value) # <- uncomment to see"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 16,
+   "id": "f92db0c1-3741-42be-9d1e-edbe5010b61e",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "\"In Rotorua, New Zealand, December falls in the Southern Hemisphere's summer, which is a great time for growing a variety of vegetables. Here are some vegetable crops you can plant in December:\\n\\n1. **Tomatoes**: Ideal for summer planting, they thrive in the warm weather.\\n2. **Capsicums (Bell Peppers)**: These also enjoy the summer heat.\\n3. **Zucchini**: Fast-growing and productive during warm months.\\n4. **Cucumbers**: Perfect for summer salads and pickling.\\n5. **Beans**: Both bush and pole beans grow well in the warm season.\\n6. **Sweet Corn**: Requires warm temperatures and plenty of sunlight.\\n7. **Pumpkins**: Plant now for a harvest in autumn.\\n8. **Eggplants**: Another heat-loving crop.\\n9. **Lettuce**: Opt for heat-tolerant varieties to avoid bolting.\\n10. **Radishes**: Fast-growing and can\""
+      ]
+     },
+     "execution_count": 16,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "response = model.invoke(prompt_value)\n",
+    "response.content"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "1ca904f4-faf1-4b95-bf77-7c7a7fa64515",
+   "metadata": {},
+   "source": [
+    "#### `ChatPromptTemplate`\n",
+    "\n",
+    "This is optimized for a conversation-like format. The prompt is a list of chat messages. Each chat message is associated with `role` and `content`. In the example code, `.from_messages` is used to include multiple messages.\n",
+    "\n",
+    "Here, we will hardcode roles in the chat prompt, as opposed to using `SystemMessage` or `HumanMessage` like earlier."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 18,
+   "id": "46e869bf-370d-4742-837f-c7d5eb76a891",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from langchain_core.prompts import ChatPromptTemplate\n",
+    "\n",
+    "chat_template = ChatPromptTemplate.from_messages(\n",
+    "    [\n",
+    "        (\"system\", \"\"\"\n",
+    "                    You are a travel agent helping customers plan their trips.\n",
+    "                    Recommend them regarding popular Accommodation, Food and Activities of the country customer is asking.\n",
```

Comment:
This may have just been cut awkwardly/copied from the markdown file incorrectly. I've changed it to:

```python
("system", """
                You're a travel agent helping customers plan their trips.
                Offer recommendations on natural features to visit, local cuisine, and activities based on the country the customer is asking about.
               """),
```